code/modules/assembly/health.dm

```diff
@@ -5,7 +5,7 @@
 	materials = list(MAT_METAL=800, MAT_GLASS=200)
 	attachable = TRUE
 
-	var/scanning = FALSE
+	var/scanning = TRUE
```

Comment:
what does changing the default for this guy fix?

Reply:
priming in hand and by assembly its two separed things for all except proxy sensors which can be turned on by the priming as qol
these things start off and you have to open a dumb giant window with only a bottom to turn them on which you are gonna do anyway
what he reported isnt a bug but intended

Reply:
kk